regolith/schemas.py

```diff
@@ -3753,38 +3769,49 @@
                     "name": {"description": "what is the deliverable of milestone",
                              "required": False,
                              "type": "string"},
-                    "notes": {"description": "small, non-deliverable to-do's",
+                    "notes": {"description": "any notes about the milestone and/or "
+                                             "small, non-deliverable to-dos to reach the milestone",
+                              "required": False,
+                              "type": "list"},
+                    "progress": {"description": "text description of progress and observations, "
+                                                "token that dereferences a figure or image in group local storage db, "
+                                                "and/or url to slides describing the development, "
+                                                "e.g., Google slides url",
                               "required": False,
                               "type": "list"},
                     "objective": {"description": "explains goal of the milestone",
                                   "required": False,
                                   "type": "string"},
-                    "audience": {"description": "rolls of audience members "
-                                                "e.g. 'pi', 'lead'",
+                    "audience": {"description": "list of people attending the meeting."
+                                                "Normally this list is group_members, collaborators, and pi, "
+                                                "or some subset of these. if people are invited who are not already"
+                                                "in these groups their names or id's can be added explicitly to the list",
                                  "required": False,
                                  "type": "list"},
-                    "status": {"description": "proposed, started, finished, converged",
+                    "status": {"description": "status of the milestone from {PROJECTUM_STATUS} "
+                                              "proposed, started, finished, converged, cancelled, paused",
                                "required": False,
-                               "type": "string",
-                               "eallowed": PROJECTA_STATUS},
-                    "type": {"description": "what kind of deliverable is it "
+                               "type": "fstring",
+                               "eallowed": PROJECTUM_STATUS},
+                    "type": {"description": "what kind of deliverable the milestone is "
```

Comment:
I am not sure if we can use fstrings in schemas here but if we can we can do it like this:
```
 "type": {"description": f"what kind of deliverable the milestone is. Allowed values are {", ".join(MILESTONE_TYPE)}"
```
Test it to  make sure it works (i.e., try and run `python run_tests.py tests/test_validate.py` from the main regolith directory.  If it does work, we should use this pattern for all the projecta entries that have ``eallowed``

Reply:
I tested it. It didn't throw up any warnings, so maybe okay?

Reply:
cool.  Go for it on all of them....

Reply:
Completed in the latest commit :)